nifi-nar-bundles/nifi-cdc/nifi-cdc-mysql-bundle/nifi-cdc-mysql-processors/src/main/java/org/apache/nifi/cdc/mysql/event/BinlogEventListener.java

```diff
@@ -29,14 +29,15 @@
 public class BinlogEventListener implements BinaryLogClient.EventListener {
 
     protected final AtomicBoolean stopNow = new AtomicBoolean(false);
-    private static final int QUEUE_OFFER_TIMEOUT_MSEC = 100;
+    private final long QUEUE_OFFER_TIMEOUT_MSEC;
```

Comment:
Let's change the variable name to a standard camel-case, `queueOfferTimeoutMsec` since it's no longer a static constant.

Reply:
Agree. Fixed!